### DISCUSSION TITLE
Design: Guardian User Personas

### DISCUSSION THREAD

**Discussion by jkitman** [fedimint/fedimint#2273] [General]
**Ben: Texas Bitcoin Enthusiast**
- **Personality:** Has a group of Bitcoiner friends, popular on Nostr, competent with technology, has setup a node before
- **Pain points:** Wants more payment privacy for himself, friends from meetups, and Nostr followers
- **Setup:** Likes to tinker with and understand the configuration options, even if other guardians (his friends) don't really care to, starts a Telegram channel to invite the friends he wants to start a Fedimint with
- **Administration:** Doesn't want any legal liability, wants to run the server for cheap, low-maintenance

**Zaire: African Community Advocate**
- **Personality:** Knows the well-respected members of her community, a few have heard of the benefits of Bitcoin but never used it
- **Goals:** Wants to enable bitcoin banking for her community which is not well served by traditional banks
- **Setup:** Invites the other guardians into her home to perform the setup, wants a simple process that will not intimidate or confuse the guardians
- **Administration:** Willing to pay out-of-pocket to keep the server running, but the grid is too unreliable to self-host and may never have setup a VPS before

**Peter: The Neo-Bank Entrepreneur**
- **Personality:** Has founded other bitcoin start-ups
- **Goals:** Wants to earn sats however possible
- **Setup:** Has a team of co-founders that he pitches the idea to, wants to configure modules and fees in order to make an income
- **Administration:** Willing to pay out-of-pocket to keep the server running, wants to constantly monitor the balance sheet of the Fedimint

**Comment by okjodom** [comment]
**Hwan: Global Activist and Bitcoin Adopter**

- **Personality**: Has a role in a globally distributed human rights organization that receives Bitcoin donations to a single address
- **Goals**: Wants to use Fedimint for more private censorship-resistant and safer custody of their Bitcoin
- **Setup**: Communicates and collaborates remotely over Signal with other guardians of their organization to set up a Fedimint. 
- **Administration**: Doesn't want any legal liability, wants to run the server for cheap, low-maintenance, and geographically distributed
